File: javaScript/just-count-sheep/scripts.js
      //just count sheep
      //let num = 0;
      function countSheep(num){
        //your code here
        arr = "";
        if(num == 0){
          arr = "";
        }
        for(i=1; i<= num; i++){
           arr = arr + `${i} sheep... `
        }
        return arr;
      }
      console.log(countSheep(0));
      console.log(countSheep(1));
      console.log(countSheep(2));
      console.log(countSheep(10));

File: dart/Is-the-string-uppercase?/scripts.dart
void main() {
  print(isUpperCase('hello I AM DONALD'));
  print(isUpperCase('HELLO I AM DONALD'));
}

bool isUpperCase(String str) {
  // your code here
  if (str == str.toUpperCase()) {
    return true;
  } else {
    return false;
  }
}
